content/blog/2019/02/2019-02-04-running-jenkins-with-java-11.adoc

```diff
@@ -0,0 +1,105 @@
+On Dec 18 (4PM UTC) we will be also presenting the Java 11 Preview Support at the Jenkins Online Meetup (link:https://www.meetup.com/Jenkins-online-meetup/events/257008190/[link]).
+
+image:/images/logos/formal/256.png[Jenkins Java, role=center, float=right]
+
+We recently made some improvement on the requirement to run Jenkins with Java 11.
+Since Jenkins core `2.163`, it is not necessary to download JAXB libraries nor add `java.xml.bind` or `java.activation` modules to the JVM. 
+This reduce the pre-requisite list to run Jenkins on Java 11.
+The JAXB libraries were extracted to the link:https://plugins.jenkins.io/jaxb[jaxb-plugin], so that plugin still requiring the libraries can use them through the plugins' dependencies mechanism.
+
+As part of the simplification, we also removed the requirement to use the Java 11 experimental update-center.
+This update center was mainly created to hold the plugins' releases which were made to try to solve issues reported specifically to Java 11 support. 
+This was the case for `workflow-support`, for example, which was solved in the release `3.1` of the plugin.
+
+=== Running Jenkins and Java 11 in Docker
+
+Starting from Jenkins `2.155`, we provide Docker images for the Jenkins master and agent.
+All these images are based on the official link:https://hub.docker.com/r/_/openjdk/[openjdk:11-jdk] image maintained by the Docker Community.
+There were discussions about migrating to other base images, but we decided to exclude it from the Preview Availability scope.
+
+==== Jenkins master image
+
+As announced previously, Java 11 support is now provided as a part of the official link:https://hub.docker.com/r/jenkins/jenkins/[jenkins/jenkins] image.
+You can run the Jenkins with Java 11 simply as:
+
+[source, shell]
+----
+docker run -p 8080:8080 -p 50000:50000 jenkins/jenkins:jdk11
+----
+
+The following tags are available:
+
+* `jdk11` - Latest weekly release with Java 11 support
+* `2.155-jdk11` - Weekly releases packaged with Java 11.
+
+NOTE: Since the weekly release `2.155`, each weekly was also released with the suffix `-jdk11`.
+So you can use `2.163-jdk11`, for example.
+
+==== Agent images
+
+If you use containerized agents via Docker or Kubernetes plugins, we have also released official Docker images for Jenkins agents:
+
+* link:https://hub.docker.com/r/jenkins/slave/[jenkins/slave]
+* link:https://hub.docker.com/r/jenkins/jnlp-slave/[jenkins/jnlp-slave]
+* link:https://hub.docker.com/r/jenkins/ssh-slave/[jenkins/ssh-slave]
+
+All images use the `latest-jdk11` image tag for JDK11 bundles.
+And sorry for the obsolete names!
+
+=== Known compatibility issues
+
+To help users to track down the compatibility issues, we have created a new link:https://wiki.jenkins.io/display/JENKINS/Known+Java+11+Compatibility+issues[Known Java 11 Compatibility Issues] Wiki page.
+
+=== Reporting compatibility issues
```

Comment:
It would be nice to have Upgrade guidelines in the blogpost. It could answer the following questions:

* What to do before the upgrade (backup, ensure all Pipelines are stopped)?
* Which plugins require an upgrade?
* How to finally replace the WAR file?
* What to do with native installers? This is actually a topic we need to discuss at the SIG meeting. Native packages are not ready yet